redhat-cop/containers-quickstarts
# Add Python 3.6 Jenkins slaves

#### What is this PR About?
Adding Dockerfiles for Python 3.6 Jenkins slaves!

#### How do we test this?
The CentOS Dockerfile can be built anywhere. The RHEL7 one only works on a RHEL host due to package manager subscription requirements. Once built, the images have `python` and `pip` set to the Python 3.6-based installations. `python --version` and `pip --version` can confirm this.

cc: @redhat-cop/containerize-it
